lighthouse-core/test/gather/driver-test.js

```diff
@@ -30,37 +27,30 @@ function createOnceStub(events) {
   };
 }
 
-function createSWRegistration(id, url, isDeleted) {
-  return {
-    isDeleted: !!isDeleted,
-    registrationId: id,
-    scopeURL: url,
-  };
+function findSendCommandInvocation(command) {
+  expect(connectionStub.sendCommand).toHaveBeenCalledWith(command, expect.anything());
+  return connectionStub.sendCommand.mock.calls.find(call => call[0] === command)[1];
 }
 
-function createActiveWorker(id, url, controlledClients, status = 'activated') {
-  return {
-    registrationId: id,
-    scriptURL: url,
-    controlledClients,
-    status,
+function createSendCommandMockFn() {
```

Comment:
docs would be good here too (e.g. the returned function has a `mockSendCommandResponse` on it, use it to push responses, can be used multiple times, where each call will get the next one, etc

Reply:
done